cmake/versions.json

```diff
@@ -1,13 +1,13 @@
 {
   "packages" : {
     "Thrust" : {
-      "version" : "1.17.0.0",
+      "version" : "2.1.0.0",
```

Comment:
Please make sure cuNumeric builds correctly with this version of Thrust. Also, I believe this version of Thrust is using the new `nv/target` headers that aren't available in CTK <= 11.5, which means this would effectively bump up the minimum CTK requirement. So I think at the very minimum the team members should be aware of this change. (I don't recall we discussed this in any of our meetings, maybe it was brought up in a devops sync-up?)

Reply:
No, we did not discuss this. Let me undo this change then, and we can discuss it on Tuesday.